node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -64,11 +67,16 @@ const LRU_OBSERVED_BLOCKS_CAPACITY: NonZeroUsize = match NonZeroUsize::new(20) {
 pub struct ScrapedUpdates {
 	pub on_chain_votes: Vec<ScrapedOnChainVotes>,
 	pub included_receipts: Vec<CandidateReceipt>,
+	pub unapplied_slashes: Vec<(SessionIndex, CandidateHash, vstaging::slashing::PendingSlashes)>,
```

Comment:
I'm not 100% convinced about adding `unapplied_slashes` to `ScrapedUpdates`. On the one hand, we're scraping the chain, so it it belongs here. OTOH, this info is only used by one function, which can embed this call there.

Reply:
Looks sensible to me to put it here at first sight.